flink/src/main/java/org/apache/iceberg/flink/RowDataTaskWriterFactory.java

```diff
@@ -127,7 +126,7 @@ private FlinkFileAppenderFactory(Schema schema, RowType flinkSchema, Map<String,
     @Override
     public FileAppender<RowData> newAppender(OutputFile outputFile, FileFormat format) {
       // TODO MetricsConfig will be used for building parquet RowData writer.
-      MetricsConfig metricsConfig = MetricsConfig.fromProperties(props);
+      // MetricsConfig metricsConfig = MetricsConfig.fromProperties(props);
```

Comment:
My only question here is why not just delete the line and todo if we aren't using them and just have a github issue instead?

Reply:
I agree here. A github issue is more likely to be picked up and worked on over a TODO in the codebase.... though I'm not opposed to keeping the TODO so people don't accidentally overly refactor away from the possibility (if that's a potential concern).

Reply:
Good point, and I fully agree. I've created a Github issue: https://github.com/apache/iceberg/issues/1402